test/e2e/common/setup-kubestellar.sh

```diff
@@ -197,4 +201,4 @@ if ! expect-cmd-output 'kubectl --context its1 get managedclusters -l location-g
 then
     echo "Failed to see two clusters."
     exit 1
-fi
+fi
```

Comment:
Please do not delete the linebreak at the end of the file. That deletion makes GitHub display an alarming character here:

<img width="204" height="110" alt="image" src="https://github.com/user-attachments/assets/ff7cc36f-7c94-477f-ac23-aa30c0ea7e54" />